reanalysis/native_slivar/slivar_classifier.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+
+
+"""
+wraps multiple slivar command submissions
+one command launched per class
+"""
+
+
+from typing import Optional
+import os
+
+import click
+import hailtop.batch as hb
+
+
+AR_REPO = 'australia-southeast1-docker.pkg.dev/cpg-common/images'
+SLIVAR_TAG = 'slivar:v0.2.7'
+SLIVAR_IMAGE = f'{AR_REPO}/{SLIVAR_TAG}'
+SLIVAR_TEMPLATE = """
+set -ex
+SLIVAR_QUIET="true" slivar expr \
+--vcf {input_vcf} \
+--pass-only \
+--skip-non-variable \
+--info '{info_expr}' \
```

Comment:
Probably not a big deal, but you could run `shlex.quote` for replacements of these parameters, in case they do happen to contain single quotes or spaces (eg: input_vcf). You'd hope not, but easy future proofing.

Reply:
I've wrapped all the command strings in a `quote`, but I don't think this slivar code is going to be used in future